core-web/package.json

```diff
@@ -211,6 +214,7 @@
         "eslint-plugin-react": "7.32.2",
         "eslint-plugin-react-hooks": "4.6.0",
         "eslint-plugin-storybook": "^0.6.15",
+        "fake-indexeddb": "^5.0.2",
```

Comment:
Not being used.

Reply:
I used in the test of the file persistence.spec.ts